packages/url_launcher/url_launcher_web/test/url_launcher_web_test.dart

```diff
@@ -110,6 +113,7 @@ void main() {
               enableDomStorage: null,
               enableJavaScript: null,
               headers: null,
+              windowName: "_blank",
```

Comment:
Revert this change, windowName should not be "_blank" by default. Also, add new tests to the `openNewWindow` group below, to test what happens when different values of windowName are passed in.